tests/DevApps/ciam/myWebApi/myWebApi.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>net7.0</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>aspnet-myWebApi-17be892d-3bb3-4282-92e2-301b02c86ed6</UserSecretsId>
+    <UseWIP>true</UseWIP>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.10" />
```

Comment:
>.0.1 [](http://example.com/codeflow?start=71&length=4)

should use the Directory.Build.Props for the test, no?

Reply:
I'll address this later (with the fact that it's still using IDownstreamWebApi. It's straight from the project templates)